### DISCUSSION TITLE
DX ToolBox, possible connection?

### DISCUSSION THREAD

**Discussion by ja-Bertolin** [AlexandreRouma/SDRPlusPlus#546] [Ideas]
Hi,

Could it be possible to connect SDR++ with DX ToolBox in order to tune frequencies from a broadcast station in DX Toolbox and activate the frequency in SDR++?

I have tried, just in case,  to configure Rigctl Server in one of the option of DXToolbox: SdxDX TCP in 4532 but there is not connection.

It could be great to be able to interact with other programs such as DXToolBox to set broadcast stations frequencies in the SDR++ like a CAT.

Thanks

**Comment by AlexandreRouma** [comment]
SDR++ supports the RigCTL protocol, I suggest asking the DX ToolBox developers to support this standard protocol.

**Reply by ja-Bertolin** [reply]
Thanks, I have just done it, they say that they can consider it at a later date, but do not have any plans to add this right now.

I will try to find an alternative compatible with RigCTL

**Reply by uxninja** [reply]
I'm also interested in this. I wonder if there are any alternatives to DX Toolbox on MacOS?
